.gitignore

```diff
@@ -0,0 +1,3 @@
+MoviewDB/node_modules/.bin/acorn
+MoviewDB/node_modules/
+MovieDB/node_modules/
```

Comment:
To many rows for a one `node_modules` folder :)  You probably need only `/node_modules` row and change file structure (more in a separate comment)

Reply:
Fixed.